alerter/build.gradle

```diff
@@ -6,14 +6,12 @@ plugins {
 apply plugin: 'com.android.library'
 apply plugin: 'maven'
 apply plugin: "maven-publish"
+apply plugin: 'com.tapadoo.android.sputnik'
```

Comment:
I agree, but it doesn't make sense for an open source project like this. It's perfect for internal work.